contrib/poll_stations.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utility script to periodically poll for station info on the local network.
+
+This script will poll for updates at a regular interval and update the terminal
+with information about currently known stations, tests, and history (in summary
+form only).
+
+This is handy for getting an at-a-glance picture of OpenHTF stations running on
+a network without having to spin up a frontend server.
+
+# TODO(madsci): Implement flags for verbosity control, logs, poll interval.
+
+Example:
+  python poll_stations.py
+
+Outputs something like, which updates regularly:
+
+Polled @05:12:44
+
+<Station madsci-mbr@192.168.1.127:8888, listening on 0.0.0.0>
+ |-- <RemoteTest MyTest(7165:1466251736061:4567501648) created 18.Sat@05:08:56, last run 18.Sat@05:08:56>
```

Comment:
eh, i'll clean this up to make it more succinct rather than just `str()`'ing the `RemoteTest`.